test/config/hosts.yml

```diff
@@ -11,6 +11,7 @@ redis_toxiproxy_port: 16379
 
 http_host: 127.0.0.1
 http_port: 31050
+http_port_2: 31150
```

Comment:
Can you think of a more descriptive name?

Reply:
Made this an array instead